google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageOptions.java

```diff
@@ -279,7 +279,8 @@ private Tuple<StorageSettings, Opts<UserProject>> resolveSettingsAndOpts() throw
         InstantiatingGrpcChannelProvider.newBuilder()
             .setEndpoint(endpoint)
             .setAllowNonDefaultServiceAccount(true)
-            .setAttemptDirectPath(attemptDirectPath);
+            .setAttemptDirectPath(attemptDirectPath)
+            .setMaxInboundMetadataSize(50 * 1024);
```

Comment:
where does this number come from?

Reply:
woops, this we left over from some test failure debugging. I'll remove it.